backend/modules/event/model.js

```diff
@@ -272,6 +273,7 @@ const EventSchema = new mongoose.Schema({
         type: Boolean,
         default: false,
     },
+    theme: { type: EventThemeSchema.mongoose, default: {} },
```

Comment:
The empty default is needed to trigger Mongoose to return the actual defaults defined in the `Schema`